src/js/recommendation-ui.js

```
@@ -6,6 +6,11 @@
  * Encapsulates the heavy analytical logic into a single async pipe.
  * Moved to outer scope to maximize reuse and minimize closure memory footprint.
  */
+
+
+pdfjsLib.GlobalWorkerOptions.workerSrc =
```

ESLint

[error] 'pdfjsLib' is not defined.

(no-undef)